test/inductor/test_triton_kernels.py

```diff
@@ -3442,12 +3441,48 @@ def decomp(*flat_args):
 
         from torch._inductor import config
 
-        with config.patch(
-            post_grad_custom_post_pass=g,
-        ):
-            f_compile = torch.compile(f)
-            self.assertEqual(f(x, other), f_compile(x, other))
-            self.assertTrue(called)
+        with torch.library._scoped_library("mylib", "FRAGMENT") as lib:
+            lib.define(
+                "force_channels_last(Tensor x) -> Tensor",
+                tags=[torch._C.Tag.flexible_layout],
+            )
+
+            def impl2(x):
+                return x.clone(memory_format=torch.channels_last)
+
+            lib.impl("force_channels_last", impl2, "CUDA")
```

Comment:
Hi, I found here is hard code "CUDA", this would fail on XPU, can we mark this test case as requires CUDA or use `GPU_TYPE.upper()` to generalize the hard code ?